.github/workflows/cdk-sam-cli-integration-testing.yml

```diff
@@ -0,0 +1,56 @@
+name: Test the integration between SAM CLI and Latest CDK version
+
+on:
+  schedule:
+    - cron: "0 13 * * 1-5" # run 1PM UTC (6 am PDT) in every weekday
+  workflow_dispatch: {}
+
+# to automatically cancel the running workflow for same PR.
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  integration-tests:
+    name:  Integration Tests / ${{ matrix.python }}
+    if: github.repository == 'aws/aws-sam-cli'
+    runs-on: ubuntu-latest
+    env:
+      AWS_DEFAULT_REGION: us-east-1
+      SAM_CLI_DEV: 1
+      NOSE_PARAMETERIZED_NO_WARN: 1
+      APPVEYOR_CONSOLE_DISABLE_PTY: false
+      APPVEYOR_DETAILED_SHELL_LOGGING: true
+    strategy:
+      matrix:
+        python:
+          - "3.7"
+          - "3.8"
+          - "3.11"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          # set last version as the one in matrix to make it default
+          python-version: |
+            3.7
+            3.8
+            3.11
```

Comment:
These are only required if we have lambda functions using these python versions in the tests. Do we have lambda functions with all these runtimes, otherwise we can remove some of them here?

That is also valid for the runtimes below, let's remove the ones that we don't use in the tests (if there are any).

Reply:
I do not remember .. let me check the lambda functions defined in the testing project

Reply:
done